src/resources/2023/BurningLeaves.ts

```diff
@@ -0,0 +1,64 @@
+import { Item, Monster, itemAmount, visitUrl } from "kolmafia";
+import { haveInCampground } from "../../lib";
+import { $item, $monster } from "../../template-string";
+import { get } from "../../property";
+
+const item = $item`A Guide to Burning Leaves`;
+
+type Thing = Item | Monster;
```

Comment:
I think this raises more questions than it answers, and I'd just put use the union explicitly when it matters